docs/server/source/drivers-clients/websocket-event-stream-api.rst

```diff
@@ -0,0 +1,105 @@
+The WebSockets Events Stream API
+================================
+
+BigchainDB provides real-time event streams over the WebSockets protocol with
+the Events Stream API.
+
+Connecting to a Events Stream from your application enables a BigchainDB node
+to notify you as events are processed, such as new `validated transactions <#valid-transactions>`_.
+
+
+Determining Support for the Events Stream API
+---------------------------------------------
+
+In practice, it's a good idea to make sure that the node you're connecting with
+has support for the Events Stream API. To do so, send a HTTP GET request to the
+node's :ref:`Root URL <bigchaindb-root-url>` and check that the response
+contains a ``streams_<version>`` property in ``_links``::
+
+    {
+      "_links": {
+         "streams_v1": "ws://example.com:9984/api/v1/streams/"
+      }
+    }
+
+
+Connection Keep Alive
```

Comment:
I added this because I didn't want the server to keep pushing updates on connections that might have died. Most WebSocket implementations require some sort of keep-alive ping/pong, although most of the time this is only necessary during periods of inactivity to avoid a firewall or etc from closing the connection prematurely.

For us, I wanted to eagerly close connections (if the client doesn't ping us back) to avoid leaving dead connections open. Not sure how easy this is to actually implement, and is something we can descope during the implementation (or remove if it's not a concern). Regardless, establishing some sort of keep-alive protocol is useful for clients that want to ensure their connection stays alive.